docs/howto/object-storage/s3/versioning.md

```diff
@@ -53,22 +108,72 @@ the following commands:
     ```bash
     aws --profile <region> \
       --endpoint-url=https://s3-<region>.{{extra.brand_domain}}:8080 \
-      s3api get-object \ 
+      s3api get-object \
       --bucket <bucket-name> \
-	  --key <object-name> \
-	  --version-id <versionid> \
-	  <local-filename>
+      --key <object-name> \
+      --version-id <versionid> \
+      <local-filename>
     ```
 === "mc"
     ```bash
     mc cp \
-	  --version-id <versionid> \
-	  <region>/<bucket-name>/<object-name> \
-	  <local-filename>
+      --version-id <versionid> \
+      <region>/<bucket-name>/<object-name> \
+      <local-filename>
     ```
 === "s3cmd"
     This functionality is not available with the `s3cmd` command.
 
 When you download an object from a versioned bucket *without*
 specifying a version identifier, your S3 client will download the
 latest version of the object that the bucket contains.
```

Comment:
I would delete the last "that" in the sentence: "that latest version of the object the bucket contains".

Reply:
Better still, trim the sentence a bit. "... the latest version of that object."